### PR TITLE
Fixes #14819: Add openssl configuration to agent package

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14819

### CODE REVIEW

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -283,6 +284,9 @@ endif
 	# Wrapper script
 	$(INSTALL) -m 755 run-inventory $(DESTDIR)/opt/rudder/bin/
 
+	# Openssl configuration
+	$(INSTALL) -m 755 openssl.cnf $(DESTDIR)/opt/rudder/etx/ssl/openssl-agent.cnf
```
Comment (amousset) [inline rudder-agent/SOURCES/Makefile]: 644 should be enough

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/SOURCES/Makefile [hunk 2]
```diff
@@ -283,6 +284,9 @@ endif
 	# Wrapper script
 	$(INSTALL) -m 755 run-inventory $(DESTDIR)/opt/rudder/bin/
 
+	# Openssl configuration
+	$(INSTALL) -m 640 openssl.cnf $(DESTDIR)/opt/rudder/etx/ssl/openssl-agent.cnf
```
Comment (amousset) [inline rudder-agent/SOURCES/Makefile]: etx -> etc

Comment (peckpeck) [reply rudder-agent/SOURCES/Makefile]: tss

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**